docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml

```diff
@@ -257,49 +255,183 @@ myelk-kibana LoadBalancer 10.110.129.199 localhost 5601:31465/TCP 68m
+      - type: "class"                              #Search by log class specific log mapping
+        searchColumn: "hpcc.log.class"             # Field containing log class information
+      - type: "instance"                           #Search by log source instance specific mapping
+        searchColumn: "kubernetes.pod.name"        # Field containing source instance information
+      - type: "host"                               #Search by log source host specific mapping
+        searchColumn: "kubernetes.node.hostname"   # Field containing source host information</programlisting>
+
+      <para>You can use the delivered Elastic4HPCCLogs chart, provided on
+      https://github.com/hpcc-systems/HPCC-Platform/tree/master/helm/managed/logging/elastic
+      or you can add the values there to your customized configuration values
+      yaml file. You can then install it using a command such as:</para>
+
+      <programlisting>helm install mycluster hpcc/hpcc -f elastic4hpcclogs-hpcc-logaccess.yaml</programlisting>
+
+      <para></para>
     </sect2>
   </sect1>
 
   <sect1 id="Azure_AKS_LogAnalytics">
-    <title>Azure Log Analytics</title>
-
-    <para>Azure Kubernetes Services (AKS) Log Analytics is an optional feature
-    designed to help monitor performance and health of Kubernetes based
-    clusters. Once enabled and associated a given AKS with an active HPCC
-    System cluster, the HPCC component logs are automatically captured by Log
-    Analytics. All STDERR/STDOUT data is captured and made available for
-    monitoring and/or querying purposes. As is usually the case with cloud
-    provider features, cost is a significant consideration and should be well
-    understood before implementation. Log content is written to the logs store
-    associated with your Log Analytics workspace.</para>
+    <title>Azure Log Analytics Solution</title>
+
+    <para>Azure Kubernetes Services (AKS) Azure Log Analytics (ALA) is an
+    optional feature designed to help monitor performance and health of
+    Kubernetes based clusters. Once enabled and associated a given AKS with an
+    active HPCC System cluster, the HPCC component logs are automatically
+    captured by Log Analytics. All STDERR/STDOUT data is captured and made
+    available for monitoring and/or querying purposes. As is usually the case
+    with cloud provider features, cost is a significant consideration and
+    should be well understood before implementation. Log content is written to
+    the logs store associated with your Log Analytics workspace.</para>
 
     <sect2>
       <title>Enabling Azure Log Analytics</title>
 
-      <para>Enabling Azure's Log Analytics on the target AKS cluster can be
-      done from the Azure portal or via CLI. For detailed Azure documentation:
-      Enable Container Log Analytics:</para>
+      <para>Enabling Azure's Log Analytics (ALA) on the target AKS cluster can
```

Comment:
Enable Azure's Log Analytics (ALA) on the target AKS cluster using one of the following options: